src/shared/sam/debugger/commands/addSamDebugConfiguration.ts

```diff
@@ -3,68 +3,82 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+import * as path from 'path'
 import * as vscode from 'vscode'
 import { LaunchConfiguration } from '../../../debug/launchConfiguration'
-import { createTemplateAwsSamDebugConfig } from '../awsSamDebugConfiguration'
+import {
+    AwsSamDebuggerConfiguration,
+    CODE_TARGET_TYPE,
+    createCodeAwsSamDebugConfig,
+    createTemplateAwsSamDebugConfig,
+    TEMPLATE_TARGET_TYPE,
+} from '../awsSamDebugConfiguration'
 import { CloudFormationTemplateRegistry } from '../../../cloudformation/templateRegistry'
 import { getExistingConfiguration } from '../../../../lambda/config/templates'
 import { localize } from '../../../utilities/vsCodeUtils'
 
 export interface AddSamDebugConfigurationInput {
     resourceName: string
-    templateUri: vscode.Uri
+    rootUri: vscode.Uri
```

Comment:
Following the field was a bit confusing, could we add some commentary around this?